consensus/height_test.go

```diff
@@ -63,7 +63,7 @@ func TestUpdateCertificate(t *testing.T) {
 	assert.Empty(t, cert.Absences())
 }
 
-func TestConsensusBehindState(t *testing.T) {
+func TestConsensusHeightIsShorterThanState(t *testing.T) {
```

Comment:
for future references might be good idea to keep pull request in titles scope and open separate PRs for typos like this (just a thought which i don't mind myself😉)